InfrastructureManager/infrastructure_manager_service.py

```diff
@@ -129,7 +129,7 @@ def stop(self):
       logging.warn('Stop called on already stopped server')
 
 if __name__ == '__main__':
-  logging.basicConfig(level=logging.INFO,
+  logging.basicConfig(level=logging.DEBUG,
```

Comment:
Is this overwritten by the incoming parameters? if not, we may want to leave it at INFO level.